serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc

```diff
@@ -33,3 +34,28 @@ To use the Red Hat Build of Quarkus (RHBQ) libraries, you need to configure your
     ...
 </project>
 ----
++
+. Remove the `quarkus-kogito-bom` dependency from the `pom.xml` file of your project:
++
+.Dependencies in `pom.xml` file
+[source,xml,subs="attributes+"]
+----
+<project>
+    ...
+    <dependencyManagement>
+        <dependencies>
+            ...
+            <dependency>
+                <groupId>io.quarkus.platform</groupId>
+                <artifactId>quarkus-kogito-bom</artifactId>
+                <version>2.7.6.Final</version>
```

Comment:
If they follow the steps, this will be 2.7.6.Final-redhat-00006.  At least currently, but it will automatically be updated if RHBQ does a new 2.7.6.x release.